### PR TITLE
Feature/add django csp

### PR DESCRIPTION
**Pull request by renejal**

### CODE REVIEW

File: dojo/settings/settings.dist.py [hunk 1]
```diff
@@ -756,12 +756,25 @@ def get_secret(secret_name):
 # Whether to use a secure cookie for the session cookie. If this is set to True,
 # the cookie will be marked as secure, which means browsers may ensure that the
 # cookie is only sent with an HTTPS connection.
-SESSION_COOKIE_SECURE = env("DD_SESSION_COOKIE_SECURE")
-SESSION_COOKIE_SAMESITE = env("DD_SESSION_COOKIE_SAMESITE")
+DD_SESSION_COOKIE_SECURE = env("DD_SESSION_COOKIE_SECURE")
```
Comment (Gaviria9601) [inline dojo/settings/settings.dist.py]: Why do you change the name of variable ?

File: dojo/settings/settings.dist.py [hunk 2]
```diff
@@ -756,12 +756,25 @@ def get_secret(secret_name):
...
+DD_SESSION_COOKIE_SAMESITE = env("DD_SESSION_COOKIE_SAMESITE")
```
Comment (Gaviria9601) [inline dojo/settings/settings.dist.py]: Why do you change the name of variable ?

File: dojo/settings/settings.dist.py (hunk 1 above)
Comment (renejal) [reply dojo/settings/settings.dist.py]: sorry, it's my mistake

File: dojo/settings/settings.dist.py (hunk 2 above)
Comment (renejal) [reply dojo/settings/settings.dist.py]: sorry, it's my mistake

**Review by Gaviria9601 — APPROVED** [review]

**Pull request merged**